Bust Config.rb

```diff
@@ -553,6 +563,12 @@ def never
     face_offset_x: 62,
     face_offset_y: 40,
   },
+  "Qum D'umpe2" => {
+    bust_scale: 0.75,
+    face_offset_x: 62,
+    face_offset_y: 40,
+    face_border_width: [0, 0, 0, 12],
```

Comment:
To make the file (slightly) easier to maintain (i.e. move all versions of Qum at once by changing a single bust_offest_x/y), might be better to use the supermode loop below and just add under it:
```rb
Busty::BASE_CONFIG["Qum D'umpe2"][:face_border_width] = [0, 0, 0, 12];
```

Reply:
Done - the border width is harmless for her main bust, so I've just applied it to both.